### PR TITLE
Add a new entry to amenity/cafe

### PR DESCRIPTION
**Pull request by iamjackslayer**

### CODE REVIEW

File: brands/amenity/cafe.json [hunk 1]
```diff
@@ -113,6 +113,19 @@
       "takeaway": "yes"
     }
   },
+  "amenity/cafe|BreadTalk": {
+    "countryCodes": ["sg"],
+    "matchNames": ["Bread Talk"],
```
Comment (bhousel) [inline brands/amenity/cafe.json]: This `matchName` is unnecessary and can be removed - the matching code ignores capitalization and spaces..

File: brands/amenity/cafe.json [hunk 2]
```diff
@@ -113,6 +113,19 @@
...
+    "tags": {
+      "amenity": "cafe",
+      "brand": "BreadTalk",
+      "brand:wikidata": "Q1106640",
+      "brand:wikipedia": "en:BreadTalk",
+      "cuisine": "coffee_shop",
```
Comment (bhousel) [inline brands/amenity/cafe.json]: I'm not familiar with this brand.. Is it definitely an `amenity=cafe` + `cuisine=coffee_shop` ?

From looking around https://www.breadtalk.com.sg/   - It looks like maybe more of a `amenity=fast_food` or `shop=bakery` ...  Cafes are really for coffee places like Starbucks.

Comment (iamjackslayer) [reply brands/amenity/cafe.json]: I stand corrected. You are right that BreadTalk is primarily a bakery. Thank you for the correction.

**Comment by bhousel** [comment]
Thank you for the quick fix, and for your first contribution!  👋

File: brands/shop/bakery.json [hunk 3]
```diff
@@ -101,6 +101,17 @@
       "shop": "bakery"
     }
   },
+  "shop/bakery|BreadTalk": {
+    "countryCodes": ["sg"],
+    "tags": {
+      "amenity": "cafe",
```
Comment (bhousel) [inline brands/shop/bakery.json]: oh I just noticed this extra `"amenity": "cafe"` still here so I removed it in eeeff003b

**Pull request merged**